dqgen/services/html_generator.py

```diff
@@ -1,17 +1,18 @@
 from jinja2 import Template
 
 from dqgen.adapters import template_builder
+from dqgen.services import QUERY_FALLBACK_LANGUAGES
 from dqgen.services.base_generator import BaseGenerator
 
 
 class HtmlGenerator(BaseGenerator):
```

Comment:
A short documentation would be advisable. How this class differs from the base, what is the purpose?